sdk/anomalydetector/ai-anomaly-detector-rest/samples-dev/batchDetectAnomalySample.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createAnomalyDetectorRestClient, {
+  BatchDetectAnomalyParameters,
+  getLongRunningPoller,
+} from "@azure-rest/ai-anomaly-detector";
+import { AzureKeyCredential } from "@azure/core-auth";
+import * as dotenv from "dotenv";
+
+dotenv.config();
```

Comment:
dotenv is imported and configured. but nowhere else in the samples is reading from `process.env`.  Consider reading the api key from environment variable.  That would also enable samples to run in build pipelines if needed.